http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -563,9 +578,12 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         raw"persistence.workspaceManager.containerResourceId=${storageContainer.resourceId.value.toString}",
 
         // identity configs
+        raw"identity.enabled=${identityType == PodIdentity}",
```

Comment:
Has the cromwhelm chart been updated with these new values?

Reply:
not yet, Janet is working on that